db/db_impl/db_impl_compaction_flush.cc

```diff
@@ -507,13 +501,25 @@ Status DBImpl::AtomicFlushMemTablesToOutputFiles(
     pick_status.push_back(false);
   }
 
+  std::vector<SequenceNumber> earliest_seqnos(num_cfs, kMaxSequenceNumber);
```

Comment:
autovector?

Reply:
it's not changed

Reply:
🤔 I think using `vector` could be better because we can allocate it in advance.